server/src/main/java/com/lunchtime/models/Restaurant.java

```diff
@@ -90,4 +86,6 @@ public Boolean isDeleted() {
     public void setDeleted(Boolean deleted) {
         isDeleted = deleted;
     }
+
+    Restaurant() {  }
```

Comment:
Constructor must be public. By Feedback to.

Reply:
https://www.javatpoint.com/access-modifiers
Default: The access level of a default modifier is only within the package. It cannot be accessed from outside the package. If you do not specify any access level, it will be the default.